qiita_plugins/target_gene/tgp/util.py

```diff
@@ -0,0 +1,187 @@
+# -----------------------------------------------------------------------------
```

Comment:
No, although I agree that some of this functionality can be used by other plugins, each plugins should be independent of Qiita. I.e. it has to be it's own repository. The only reason this is currently in the same repository is for convenience. The reason for this is because different plugins are going to have different requirements. E.g. The Unifrac plugin will use the latest skbio while qiita will be still requiring 0.2.3 due to this plugin.